onnxruntime/core/providers/nv_tensorrt_rtx/nv_execution_provider.h

```diff
@@ -266,6 +268,7 @@ class NvExecutionProvider : public IExecutionProvider {
   std::string cache_prefix_;
   std::string op_types_to_exclude_;
   int nv_profile_index_ = 0;
+  std::vector<std::unique_ptr<onnxruntime::Model>> ep_context_models_;
```

Comment:
>ep_context_models_ [](http://example.com/codeflow?start=51&length=18)

The model instance here is just help to hold the EPContext nodes. Keep one instance is enough, why need a list of models?